Sofa/Component/IO/Mesh/src/sofa/component/io/mesh/STLExporter.cpp

```diff
@@ -137,24 +137,24 @@ bool STLExporter::writeSTL(bool autonumbering)
     }
     if(!triangleIndices.empty())
     {
-        for(unsigned int i=0;i<triangleIndices.size();i++)
+        for(auto triangleIndice : triangleIndices)
```

Comment:
```suggestion
        for(const auto& triangleIndex : triangleIndices)
```

const ref should be usable

(this applies to the other changes in STLExporter, lazy to do the other suggestions 😅)

Reply:
if I am not wrong, `triangleIndice` is a primitive type so it's preferable to pass by copy, no?

Reply:
ah no, my bad, it's a vector of triangle, not an array of indices.